handbook/engineering/leadership/index.md

```diff
@@ -0,0 +1,80 @@
+
+- Problem: [We decided to change how we think about goals](https://github.com/sourcegraph/about/pull/1177).
+- Plan: Since we are no longer doing OKRs, we need each team to create an initial set of [goals](../../../company/goals.md) to communicate what they are working on and why. This is done when each team has documented an initial set of goals that Nick and Christina approve.
+- Owner: Each EM owns this for their own team and should communicate the status of the goal setting process for their team in the handbook itself.
+- Status: [In progress](../../../company/goals.md#engineering).
+
+#### Career guidance for engineering roles at Sourcegraph
+
+- Problem: It is not clear to engineers on our team how we think about career development at Sourcegraph.
+- Plan: Write a blog post that illustrates possible career paths.
+- Owner: Chris
+- Status: [In progress](https://docs.google.com/document/d/1ARfJ9Pg8Z9ZbAj9VsN8VLfaR2b4_akDn9k2NMy4NmLM/edit).
+
+#### Fullstack engineer role
+
+- Problem: Cloud and Code Intelligence teams have a mix of frontend and backend work, but current workload is skewed toward the latter. It doesn't make sense to hire a single frontend dev because they would end up working alone and there might not be enough work to sustain a full time frontend dev.
+- Plan: We are defining a new fullstack role and interview process to help us find engineers who can do the necessary frontend work, but also contribute to our backend systems. We think this role could be valuable for other teams, like Search and Campaigns, in the future. 
+- Owners: Tomás, Aida
+- Status: [In progress](https://github.com/sourcegraph/interviews/pull/136).
+
+#### Engineering manager onboarding
+
+- Problem: We recently hired and onboarded our first group of engineering managers and onboarded each of them in an ad hoc way. We will be hiring at least one more engineering manager in the near future (for the web team) and we want to document what we have learned so they have an even better onboarding experience.
+- Owner: Nick
+- Plan: Document the onboarding process for engineering managers in our handbook before the EM for the web team starts.
+- Status: Not started.
+
+### Medium term (~3-6 months)
+
+#### Publicly documented compensation bands
+
+- Problem: When we were smaller, it was easy to contain consistent compensation across the entire team because there were fewer people to coordinate with. Now that we are bigger and have more engineering managers making offers and compensation decisions, we need to have clearly documented compensation bands so we can maintain consistency while federating decision making. We want to make this public because it will help candidates understand what they can expect before they even apply. This is a better candidate experience and it avoid rare but real situations where a candidate goes through our interview process only to discover that their compensation expectations don't fit into our bands.
```

Comment:
s/avoid/avoids/

Reply:
Thanks! Fixed in 35c018d22209f2625d23b8ec330ad56699b66352